controls/cis_rhel9.yml

```diff
@@ -823,7 +823,10 @@ controls:
     levels:
       - l1_server
       - l1_workstation
-    status: planned
+    status: automated
+    rules:
+      - package-dnsmasq-removed
```

Comment:
```suggestion
      - package_dnsmasq_removed
```

Reply:
The rule id is basically the folder name where the respective `rule.yml` is located. I believe you already know it, but in any case. ; )

Reply:
Thank you for catching it :)